src/it/scala/org/zalando/spearheads/innkeeper/routes/PostPathsSpec.scala

```diff
@@ -224,6 +249,18 @@ class PostPathsSpec extends FunSpec with BeforeAndAfter with Matchers {
           entityString(response).parseJson.convertTo[Error].errorType should be("ITE")
         }
       }
+
+      describe("when a token without admin privileges is provided when creating a regex path") {
```

Comment:
This test is for a token without admin privileges. A new test isn't needed for the fail path. One more test is needed for the happy path.